### PR TITLE
[MM-44858] Screen reader fails in announcing no result found in dm modal

### PR DESCRIPTION
**Pull request by babinderrathi**
#### Summary
Screen reader fails in announcing No result found in dm modal.
https://mattermost.atlassian.net/browse/MM-44858

#### Ticket Link

  Fixes https://mattermost.atlassian.net/browse/MM-44858
 Fixes https://github.com/mattermost/mattermost-server/issues/21260

#### Related Pull Requests
None

#### Screenshots
|  before  |

https://user-images.githubusercontent.com/102028591/190338880-68cbabea-76e2-48f8-8a1c-8dc63dd5e91e.mp4

|  after  |

https://user-images.githubusercontent.com/102028591/190334214-97253e31-12e9-4e83-941b-37908a4e186a.mp4

#### Release Note
```release-note
 In the Direct Messages modal (+ next to Direct Messages) when we try to search anything which is not present, a message ( No results found matching)  is visible and not accessible by the screenreaders. 
```

### CODE REVIEW

File: components/multiselect/multiselect_list.tsx [hunk 1]
```diff
@@ -185,13 +186,15 @@ export default class MultiSelectList<T extends Value> extends React.PureComponen
                     <div
                         key='no-users-found'
                         className='no-channel-message'
+                        tabIndex={0}
                     >
                         <p className='primary-message'>
-                            <FormattedMarkdownMessage
+                            <FormattedMessage
```
Comment (hmhealey) [inline components/multiselect/multiselect_list.tsx:192]: Thanks for changing that as well!

**Review by hmhealey — APPROVED** [review]
This code looks good to me, but I want to check with @esethna to make sure we aren't expecting the text to get read out automatically. I suppose that the screen reader won't read out that there are results without the user tabbing to the list, so this matches that behaviour more or less.

**Comment by esethna** [comment]
Moving this PM review to @laneycs.

**Comment by esethna** [comment]
@laneycs see Harrisons comment above:

> This code looks good to me, but I want to check with @esethna to make sure we aren't expecting the text to get read out automatically. I suppose that the screen reader won't read out that there are results without the user tabbing to the list, so this matches that behaviour more or less.

**Review by laneycs — APPROVED** [review]

**Review by pvev — APPROVED** [review]
LGTM! thanks @babinderrathi 👍

**Comment by AshishDhama** [comment]
@jgilliam17 Gentle reminder on this.

**Review by jgilliam17 — APPROVED** [review]
Thank you @babinderrathi 
Tested, looks good to merge.
Verified screen reader announces no results found
[E2E report](https://mattermost-cypress-report.s3.amazonaws.com/249658-46f00fa-onprem-ent-webapp-pr-11140-mm-ee-test:46f00fa/mochawesome.html) - no PR related failures.

**Pull request merged**